tests/integration/targets/connection_proxmox_pct_remote/dependencies.yml

```diff
@@ -0,0 +1,18 @@
+---
+# Copyright (c) 2025 Nils Stein (@mietzen) <github.nstein@mailbox.org>
+# Copyright (c) 2025 Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- hosts: localhost
+  gather_facts: true
+  serial: 1
+  tasks:
+    - name: Copy pct mock
+      copy:
+        src: files/pct
+        dest: /usr/sbin/pct
+        mode: '0755'
+    - name: Install paramiko
```

Comment:
I thought paramiko would already be part of Ansible, but not at the computer now, will check later

Reply:
That used to be the case, but is no longer true: https://github.com/ansible/ansible/blob/devel/requirements.txt
(It was removed in 2019: https://github.com/ansible/ansible/commit/6ce9cf7741679449fc3ac6347bd7209ae697cc5b#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552)